### ISSUE TITLE
Data list has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1934]
- id attribute value must be unique (seems to be issue with headings)

**Comment by redallen** [comment]
Closing in favor of #2158.
